application/comit_node/src/swap_protocols/rfc003/ethereum/ether_htlc.rs

```diff
@@ -49,7 +49,7 @@ impl EtherHtlc {
         let n_bytes = bytes.0.len();
         let gas_per_byte = 200;
 
-        U256::from(50_000 + n_bytes * gas_per_byte)
+        U256::from(75_000 + n_bytes * gas_per_byte)
```

Comment:
❓  Why the update?

Reply:
The value returned by this function was already not being used in the BTC->ETH e2e test, because it was slightly too low. So I changed it so that we can use it both for ETH->BTC and BTC->ETH (I changed all this as part of [this small commit](https://github.com/comit-network/comit-rs/pull/608/commits/09c2c55ba67196ffa4b9812e13da3f53fd5de3da).

Reply:
Nice!